xgboost_ray/main.py

```diff
@@ -605,6 +604,11 @@ def predict(self, model: xgb.Booster, data: RayDMatrix, **kwargs):
         return predictions
 
 
+@ray.remote
+class RayXGBoostActor(_RayXGBoostActor):
```

Comment:
maybe the docs should go here instead, since this one is more public facing?

Reply:
They will be inherited, right?